string name
uint8 type
float64 x
float64 y
float64 z
float64 eeo
float64 gripper_rot
float64 gripper_lin
uint8 velocity
uint8 precision
uint16 smoothness
uint16 tolerance
string shape
uint8 tag_id
uint16 servo_one
uint16 servo_two
uint16 servo_three
uint16 servo_four
uint16 servo_five
uint16 servo_six



